### ISSUE TITLE
Performance gap with Llama-2-7B

### ISSUE THREAD

**Issue by Xzk7** [OpenGVLab/OmniQuant#94]
Sorry to bother you, I run the weight-only quantization evaluation with the command below:

CUDA_VISIBLE_DEVICES=0 python main.py \
--model /PATH/TO/LLaMa/Llama-2-7b --eval_ppl --net Llama-2-7b \
--epochs 0 --output_dir ./log/Llama-2-7b-w3a16g128test \
--wbits 3 --abits 16 --group_size 128 --lwc --resume log/llama-2-7b-w3a16g128/Llama-2-7b-w3a16g128.pth

But I encounter the problem that Attention mask size mismatch error in Line156 in init_llama_layer.py, then I change "attention_mask=attention_mask" in Line247 with "attention_mask=attention_mask[..., :-1]", the performance is:

INFO wikitext2 : 6.209373950958252
INFO c4 : 7.872798919677734

a bit higher than the results in paper, could you help me with this issue? Thank you!

**Comment by ChenMnZ** [comment]
The checkpoints have some mismatch with current code.

Retrain by yourself through current code can successfully reproduce the results.
